Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/negative/SubmissionTestResultNegativeFragment.kt

```diff
@@ -169,6 +163,14 @@ class SubmissionTestResultNegativeFragment : Fragment(R.layout.fragment_submissi
         }
     }
 
+    private fun setNegativeResultStepOrRemoveTestStepAsFinal(coronaTest: BaseCoronaTest) {
```

Comment:
The naming is confusing to me, it is about the no pending certificate request, or not?